openedx/core/djangoapps/discussions/models.py

```diff
@@ -609,6 +609,10 @@ class DiscussionTopicLink(models.Model):
         default=True,
         help_text=_("Whether this topic should be shown in-context in the course.")
     )
+    ordering = models.PositiveIntegerField(
```

Comment:
do you think you need to define a default value here for the rows that exist? or there aren't any rows in this model at the moment? 
I say this because this is a non-nullable value without any default value and it can cause issues.

Reply:
I think you missed the `null=True` parameter on the next line. This is a nullable field so the default value will be null.